runtime/src/swam/runtime/import_generator/ImportGenerator.scala

```diff
@@ -0,0 +1,182 @@
+package swam.runtime.import_generator
```

Comment:
For a reason I cannot really explain, I really dislike underscores in package names, as well as camel case. I like my packages to be one word, or if it is really necessary several words, all lower case. In this case I think `swam.runtime.generator` is an ok name.